set.mm

```diff
@@ -633292,21 +633570,539 @@ fixed reference functional determined by this vector (corresponding to
   $}
 
   ${
-    lcmfunnnd.1 $e |- ( ph -> N e. NN ) $.
-    $( Useful equation to calculate the least common multiple of 1 to n.
-       (Contributed by metakunt, 29-Apr-2024.) $)
-    lcmfunnnd $p |- ( ph -> ( _lcm ` ( 1 ... N ) ) =
-      ( ( _lcm ` ( 1 ... ( N - 1 ) ) ) lcm N ) ) $=
-      ( c1 cfz co clcmf cfv cmin csn cun cuz wcel wceq cc0 cn0 syl eleq2i a1i
-      cz clcm caddc nncnd npcand oveq2d cn nnm1nn0 nn0uz sylib wb fveq2i mpbird
-      1cnd 1m1e0 fzsuc2 mpan eqtr3d sneqd uneq2d eqtrd fveq2d wss cfn w3a fzssz
-      1z fzfi nnz 3jca lcmfunsn ) ADBEFZGHDBDIFZEFZBJZKZGHZVMGHBUAFZAVKVOGAVKVM
-      VLDUBFZJZKZVOADVREFZVKVTAVRBDEABDABCUCAUMUDZUEAVLDDIFZLHZMZWAVTNZAWEVLOLH
-      ZMZAVLPMZWHABUFMZWICBUGQPWGVLUHRUIWEWHUJAWDWGVLWCOLUNUKRSULDTMWEWFVFDVLUO
-      UPQUQAVSVNVMAVRBWBURUSUTVAAVMTVBZVMVCMZBTMZVDVPVQNAWKWLWMWKADVLVESWLADVLV
-      GSAWJWMCBVHQVIBVMVJQUT $.
+    $d K k $.  $d M k $.  $d N k $.
+    lcmineqlem4.1 $e |- ( ph -> N e. NN ) $.
+    lcmineqlem4.2 $e |- ( ph -> M e. NN ) $.
+    lcmineqlem4.3 $e |- ( ph -> M <_ N ) $.
+    lcmineqlem4.4 $e |- ( ph -> K e. ( 0 ... ( N - M ) ) ) $.
+    $( Part of lcm inequality lemma, this part eventually shows that F times
+       the least common multiple of 1 to n is an integer.  (Contributed by
```

Comment:
Is the comment correct? Where is the ` F ` in the theorem?

Reply:
It is referenced now